### PR TITLE
Add tutorial 6

### PR DESCRIPTION
**Pull request by devvoid**
Took a while due to holiday business and recent illness, but I've finally got tutorial 6 almost finished. Hopefully, tutorials 7 and 8 shouldn't take as long.

### CODE REVIEW

File: learn/chapter1/6-transformations.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+## Vectors
+
+In its most basic definition, vectors are directions and nothing more. A vector has a direction and a magnitude (also known as its strength or length). You can think of vectors like directions on a treasure map: 'go left 10 steps, now go north 3 steps and go right 5 steps'; here 'left' is the direction and '10 steps' is the magnitude of the vector. The directions for the treasure map thus contains 3 vectors. Vectors can have any dimension, but we usually work with dimensions of 2 to 4. If a vector has 2 dimensions it represents a direction on a plane (think of 2D graphs) and when it has 3 dimensions it can represent any direction in a 3D world.
+
+Below you'll see 3 vectors where each vector is represented with (x,y) as arrows in a 2D graph. Because it is more intuitive to display vectors in 2D (than in 3D) you can think of the 2D vectors as 3D vectors with a z coordinate of 0. Since vectors represent directions, the origin of the vector does not change its value. In the graph below we can see that the vectors \(\bar{v}\)
+and \(\bar{w}\) are equal even though their origin is different:
```
Comment (DylanFPS) [inline learn/chapter1/6-transformations.md]: `\(\bar{w}\)` - this and the v equivalent didn't translate into markdown

File: learn/chapter1/6-transformations.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+
+![Vectors](img/6-vectors.png)
+
+When describing vectors mathematicians generally prefer to describe vectors as character symbols with a little bar over their head like \(\bar{v}\). Also, when displaying vectors in formulas they are generally displayed as follows:
+
+\[\bar{v} = \begin{pmatrix} \color{red}x \\ \color{green}y \\ \color{blue}z \end{pmatrix} \]
```
Comment (DylanFPS) [inline learn/chapter1/6-transformations.md]: The entirety of this bit didn't translate into markdown.

File: learn/chapter1/6-transformations.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+Because vectors are specified as directions it is sometimes hard to visualize them as positions. What we basically visualize is we set the origin of the direction to **(0,0,0)** and then point towards a certain direction that specifies the point, making it a position vector (we could also specify a different origin and then say: 'this vector points to that point in space from this origin'). The position vector **(3,5)** would then point to **(3,5)** on the graph with an origin of **(0,0)**. Using vectors we can thus describe directions and positions in 2D and 3D space.
+
+Just like with normal numbers we can also define several operations on vectors (some of which you've already seen).
+
+## Scalar vector operations
+
+A scalar is a single digit (or a vector with 1 component if you'd like stay in vector-land). When adding/subtracting/multiplying or dividing a vector with a scalar we simply add/subtract/multiply or divide each element of the vector by the scalar. For addition it would look like this:
+
+\[ \begin{pmatrix} \color{red}1 \\ \color{green}2 \\ \color{blue}3 \end{pmatrix} + x = \begin{pmatrix} \color{red}1 + x \\ \color{green}2 + x \\ \color{blue}3 + x \end{pmatrix} \]
```
Comment (DylanFPS) [inline learn/chapter1/6-transformations.md]: You get the point.

File: learn/chapter1/6-transformations.md [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+  
+Where \(+\) can be \(+\),\(-\),\(\cdot\) or \(\div\) where \(\cdot\) is the multiplication operator. Keep in mind that for the \(-\) and \(\div\) operator order, the reverse order is not defined.
+
+## Vector negation
+
+Negating a vector results in a vector in the reversed direction. A vector pointing north-east would point south-west after negation. To negate a vector we add a minus-sign to each component (you can also represent it as a scalar-vector multiplication with a scalar value of -1):
+
+\[-\bar{v} = -\begin{pmatrix} \color{red}{v_x} \\ \color{blue}{v_y} \\ \color{green}{v_z} \end{pmatrix} = \begin{pmatrix} -\color{red}{v_x} \\ -\color{blue}{v_y} \\ -\color{green}{v_z} \end{pmatrix} \]
```
Comment (DylanFPS) [inline learn/chapter1/6-transformations.md]: Didn't render.

File: learn/chapter1/6-transformations.md [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+
+## Addition and subtraction
+
+Addition of two vectors is defined as component-wise addition, that is each component of one vector is added to the same component of the other vector like so:
+
+\[\bar{v} = \begin{pmatrix} \color{red}1 \\ \color{green}2 \\ \color{blue}3 \end{pmatrix}, \bar{k} = \begin{pmatrix} \color{red}4 \\ \color{green}5 \\ \color{blue}6 \end{pmatrix} \rightarrow \bar{v} + \bar{k} = \begin{pmatrix} \color{red}1 + \color{red}4 \\ \color{green}2 + \color{green}5 \\ \color{blue}3 + \color{blue}6 \end{pmatrix} = \begin{pmatrix} \color{red}5 \\ \color{green}7 \\ \color{blue}9 \end{pmatrix} \]
```
Comment (DylanFPS) [inline learn/chapter1/6-transformations.md]: Didn't render.

File: learn/chapter1/6-transformations.md [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+
+Visually, it looks like this on vectors **v=(4,2)** and **k=(1,2)**:
+
+![Addition of vectors](img/6-vector_addition.png)
+
+Just like normal addition and subtraction, vector subtraction is the same as addition with a negated second vector:
+
+\[\bar{v} = \begin{pmatrix} \color{red}1 \\ \color{green}2 \\ \color{blue}3 \end{pmatrix}, \bar{k} = \begin{pmatrix} \color{red}4 \\ \color{green}5 \\ \color{blue}6 \end{pmatrix} \rightarrow \bar{v} + -\bar{k} = \begin{pmatrix} \color{red}1 + (-\color{red}{4}) \\ \color{green}2 + (-\color{green}{5}) \\ \color{blue}3 + (-\color{blue}{6}) \end{pmatrix} = \begin{pmatrix} -\color{red}{3} \\ -\color{green}{3} \\ -\color{blue}{3} \end{pmatrix} \]
```
Comment (DylanFPS) [inline learn/chapter1/6-transformations.md]: Didn't render

File: learn/chapter1/6-transformations.md [hunk 7]
```diff
@@ -0,0 +1,359 @@
...
+
+![Subtraction of vectors](img/6-vector_subtraction.png)
+
+## Length
+
+To retrieve the length/magnitude of a vector we use the Pythagoras theorem that you might remember from your math classes. A vector forms a triangle when you visualize its individual **x** and **y** component as two sides of a triangle:
+
+![Pythagoras theorem](img/6-vector_triangle.png)
+
+\[||\color{red}{\bar{v}}|| = \sqrt{\color{green}x^2 + \color{blue}y^2} \]
```
Comment (DylanFPS) [inline learn/chapter1/6-transformations.md]: Yeah none of this renders. :|

**Comment by devvoid** [comment]
How can I embed Latex into Markdown? I've never done anything like this before (it renders fine in vscode's preview)

**Comment by devvoid** [comment]
Replaced the first few bits of LaTeX with screenshots from the original article. Want to be sure this is good before I move on to the rest.

**Comment by varon** [comment]
Anything is fine. You can try copy the raw HTML from the other website as an alternative.

**Comment by DylanFPS** [comment]
New changes look good.

**Pull request merged**